ocaml/idl/datamodel_errors.ml

```diff
@@ -177,7 +177,7 @@ let _ =
     () ;
 
   (* Session errors *)
-  error Api_errors.session_authentication_failed []
+  error Api_errors.session_authentication_failed ["username"; "msg"]
```

Comment:
Please update all the users of the error to use the new parameters, I found at least two out of date ones:
- https://github.com/xapi-project/xen-api/blob/master/ocaml/xapi/xapi_http.ml#L264
- https://github.com/xapi-project/xen-api/blob/master/ocaml/tests/test_session.ml#L12